src/translations/activateTranslation.ts

```diff
@@ -0,0 +1,14 @@
+import { i18n } from "@lingui/core"
+
+i18n.loadLocaleData({
+  en : {},
+  nl: {},
+  pseudo: {},
+})
```

Comment:
You should have plural rules

Reply:
Will re-add, thought they were optional (as it did not break anything nor produced errors in console).

Reply:
you can use this snippet 

```ts
export function activateLocale(i18n: I18n, locale: string, messages: Messages) {
  const cardinalRules = new Intl.PluralRules(locale, { type: 'cardinal' });
  const ordinalRules = new Intl.PluralRules(locale, { type: 'ordinal' });

  i18n.loadLocaleData(locale, {
    plurals: (n, ordinal) => {
      return (ordinal ? ordinalRules : cardinalRules).select(n);
    },
  });

  i18n.load(locale, messages)
  i18n.activate(locale)
}
```
from here https://github.com/thekip/nextjs-translation-demo/blob/update-for-latest-lingui/src/utils.ts

Reply:
Without plural rules plurals will not work correctly in languages diffrent from english. read here https://lingui.dev/guides/plurals